src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -32,6 +32,10 @@ public class HttpClientHandlerTest
 
         private readonly NetworkCredential _credential = new NetworkCredential(Username, Password);
 
+        public readonly static object[][] GetServers = HttpTestServers.GetServers;
```

Comment:
Xunit demands that they be public. Must have something to do with how they use reflection.

This is what happens when they are not public:

`
    System.Net.Http.Functional.Tests.HttpClientHandlerTest.PostAsync_CallMethod_NullContent [FAIL]
        System.ArgumentException : Could not find public static member (property, field, or method) named 'PostServers' on System.Net.Http.Functional.Tests.HttpClientHandlerTest
`